### DISCUSSION TITLE
Semantic HTML5 layout

### DISCUSSION THREAD

**Discussion by uPagge** [umputun/remark42#918] [Ideas]
I would like to suggest adding a semantic layout for your comments.

An example can be found in the specification in this section:

https://html.spec.whatwg.org/multipage/sections.html#the-article-element

**Comment by paskal** [comment]
Good idea! I would love to see someone submitting a PR that would add proper meta to the comments, I would suggest (also) looking at [shema.org Comment](https://schema.org/Comment) type.

As far as I can see, comments [are already wrapped in the article](https://github.com/umputun/remark42/blob/e1e8da4b2aa3f79a47149bae75f13dbaf6c221d3/frontend/app/components/comment/comment.tsx#L605) type so adding metainformation to them should be fairly trivial.

**Comment by akellbl4** [comment]
I'm keeping it in mind for our design update. I'll work on it soon and I'll start from the comment form. But if someone can do it with the current markup it would be great.
I think we can convert the discussion into an issue.
